resources/views/exchanges/dashboard/proposed/empty.blade.php

```diff
@@ -0,0 +1,10 @@
+<div class="card-body p-5 text-center">
+    @if(Auth::user()->isAdmin())
+        <h4 class="card-title"><small>No proposed exchanges.</small></h4>
+        <p class="card-text text-muted">This user hasn't proposed any shift exchange so far.</p>
```

Comment:
If the user is an `admin`, shouldn't the message be a little different?
This indicates that an `admin` can exchange shifts, and that's not true, right?